quickstart/how_to_start.md

```diff
@@ -33,35 +33,34 @@ __Config__ — a `config.json` file, a part of a Feature. Configs from all loade
 
 __Map__, __Message Map__, __Chain__ — the list of actors in the specified order to process a message step by step. On each step the message is wrapped and passed to the Actor's handler, the Actor makes some modifications in the message atomically, then the message is passed to the next Actor. This is the core idea, how independent Actors work together.
 
-__Plugin__ — a part of a Feature, contains code to initialize the Feature, in most cases registers new strategies in IOC. Plugins may depend on other plugins.
+__Plugin__ — a part of a Feature, contains code to initialize on feature loading, in most cases registers new strategies in IOC.
```

Comment:
It can, but:
1. Info about it should not belong here, it should be in the section about plugins
2. It's not recommended, since we're basically hard wiring plugins in code. For these purposes we usually use `afterFeatures` section in the feature config